ros_ign_bridge/test/subscribers/ign_subscriber.cpp

```diff
@@ -211,6 +211,18 @@ TEST(IgnSubscriberTest, TransformStamped)
   EXPECT_TRUE(client.callbackExecuted);
 }
 
```

Comment:
Should we add a double test here?

Reply:
I added one but there was none present in the melodic branch so maybe one should be added there too.

Reply:
Thanks! And good catch. Yeah ideally we'd have one there too.